pkgs/tools/inputmethods/hime/default.nix

```
@@ -16,25 +17,18 @@ stdenv.mkDerivation {
   };
 
   nativeBuildInputs = [ which pkgconfig ];
-  buildInputs = [ libXtst gtk2 gtk3 qt4 qt5.qtbase ]
-    ++ lib.optional enableChewing libchewing
-    ++ lib.optional enableAnthy anthy;
-
-  patchPhase = ''
-    patchShebangs configure
-  '';
+  buildInputs = [ libXtst gtk2 gtk3 qt4 qt5.qtbase ];
 
-  # The configure script already auto-detect libchewing and anthy,
-  # so we do not need additional flags for libchewing or anthy
+  patchPhase = "patchShebangs configure";
```

we usually don't override patchPhase, because it might collide with the `patches` attribute
```suggestion
  preConfigure = "patchShebangs configure";
```
besides that, LGTM

---

Many thanks.